Sources/AppStoreConnectCLI/Model/Profile.swift

```diff
@@ -2,33 +2,23 @@
 
 import AppStoreConnect_Swift_SDK
 import Foundation
+import Model
 import SwiftyTextTable
 
-struct Profile: Codable {
-    var name: String?
-    var platform: BundleIdPlatform?
-    var profileContent: String?
-    var uuid: String?
-    var createdDate: Date?
-    var profileState: ProfileState?
-    var profileType: ProfileType?
-    var expirationDate: Date?
-}
-
-extension Profile {
+extension Model.Profile {
     init(_ apiProfile: AppStoreConnect_Swift_SDK.Profile) {
         self.init(apiProfile.attributes!)
     }
 
     init(_ attributes: AppStoreConnect_Swift_SDK.Profile.Attributes) {
         self.init(
             name: attributes.name,
-            platform: attributes.platform,
+            platform: attributes.platform?.rawValue,
             profileContent: attributes.profileContent,
             uuid: attributes.uuid,
             createdDate: attributes.createdDate,
-            profileState: attributes.profileState,
-            profileType: attributes.profileType,
+            profileState: attributes.profileState?.rawValue,
+            profileType: attributes.profileType?.rawValue,
```

Comment:
For enum variable like this, it will be alright for fetching from the server and convert to string. But there might need an extra reconvert if we want to read data model from local files (eg. read string data and convert back to enum for API requesting).

Reply:
Yeah there will need to be some enum handling
The plan will probably be to declare our own enums in `Model` and then to house all the existing model conversions in `Network`
Even though I don't really enjoy the duplication